ogr/ogrsf_frmts/hana/ogrhanalayer.cpp

```diff
@@ -765,41 +765,69 @@ OGRErr OGRHanaLayer::InitFeatureDefinition(const CPLString &schemaName,
 /*                         ReadGeometryExtent()                         */
 /************************************************************************/
 
-void OGRHanaLayer::ReadGeometryExtent(int geomField, OGREnvelope *extent)
+void OGRHanaLayer::ReadGeometryExtent(int geomField, OGREnvelope *extent,
+                                      int force, bool fastExtentMethod)
 {
     EnsureInitialized();
 
     OGRGeomFieldDefn *geomFieldDef = featureDefn_->GetGeomFieldDefn(geomField);
+    if (geomFieldDef == nullptr)
```

Comment:
This is already checked in OGRHanaLayer::GetExtent.

Reply:
Done.